cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/lineage/field/FieldLineageInfo.java

```diff
@@ -338,6 +343,12 @@ private long computeChecksum() {
                                      operationsMap.get(input.getOrigin()), write, summary);
       }
     }
+    for (String field : droppedFields.keySet()) {
```

Comment:
This is NOT what I mean, you shouldn't rely on the `fieldSources`, I told you in the comments that this is not a correct way to do that. It should have something here like the writeOperations,
```
for (TransformOperation dropped: droppedOperations) {
      List<InputField> inputs = dropped.getInputs();
      for (InputField input : inputs) {
        computeIncomingSummaryHelper(...);
      }
    }
```